example-notebooks/ocean-npp-timeseries-analysis.ipynb

```diff
@@ -0,0 +1,1267 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        print(item['properties']['start_datetime'])
</pre><p>Nitpick:  but I think it would be better to use a month name and year rather than an iso string. Something like: <code>datetime(item["properties"]["start_datetime"]).strftime("%B %Y")</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
This makes sense and a good call. I've revised the PR to reflect this and it is much more readable now. 👍